packages/hub/src/app/users/[username]/models/[slug]/relative-values/[variableName]/CacheMenu/index.tsx

```diff
@@ -30,18 +38,57 @@ export const CacheMenu: FC<{
 
   const isEmpty = revision.forRelativeValues.cache.length === 0;
 
-  return (
+  const fullyCached =
+    !isEmpty &&
+    revision.forRelativeValues.cache.length >=
+      definition.items.length * definition.items.length;
+
+  const ownedByCurrentUser = ownerUsername === session?.user?.username;
+
+  const internals = (
+    <div
+      className={clsx(
+        "flex items-center text-sm text-gray-500 px-2 py-1 rounded-sm",
+        ownedByCurrentUser && "hover:bg-slate-200 cursor-pointer"
+      )}
+    >
+      {fullyCached ? (
+        <>
+          <CheckIcon className="text-gray-600" />
+          Cached
+        </>
+      ) : (
+        <>
+          <XIcon size={12} className="mr-1" />
+          Not Cached
+        </>
+      )}
+    </div>
+  );
+
+  let withDropdown = (internals: ReactElement) => (
     <Dropdown
       render={({ close }) => {
         if (!revision.forRelativeValues?.id) {
           return null; // shouldn't happen, this is mostly for type safety
         }
         return (
           <DropdownMenu>
-            <BuildRelativeValuesCacheAction
-              exportId={revision.forRelativeValues.id}
-              close={close}
-            />
+            <DropdownMenuHeader>
+              {" "}
```

Comment:
(Minor)
I've noticed sometimes you write `<Header> Foo </Header>` instead of `<Header>Foo</Header>`, and then after refactorings it turns into these noisy spaces.

Reply:
Oh, thanks for flagging!